pkg/controllers/monitoring/monitoring-stack/components.go

```diff
@@ -52,6 +52,7 @@ func stackComponentReconcilers(
 	alertmanagerName := ms.Name + "-alertmanager"
 	additionalScrapeConfigsSecretName := ms.Name + "-self-scrape"
 	hasNsSelector := ms.Spec.NamespaceSelector != nil
+	createCRB := hasNsSelector && ms.Spec.CreateClusterRoleBindings == "CreateClusterRoleBindings"
```

Comment:
(nit) we could use `CreateClusterRoleBindings`?

Reply:
Yes good point! Fixed.